### PR TITLE
Created footer component and styling

### PR DESCRIPTION
**Pull request by francescastocco**
https://trello.com/c/IH1dqIwP

### CODE REVIEW

**Comment by francescastocco** [comment]
not ready for review - need to add tests

File: WhaleSpotting/react-app/src/tests/Footer.test.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { Footer } from "../components/Footer";
+import React from "react";
+
+test("renders Whale Museum logo in footer", () => {
+    render(<Footer />);
+
+    const whaleMuseumLogo = screen.getByTestId("whale-museum-logo");
+
+    expect(whaleMuseumLogo).toBeInTheDocument();
+    expect(whaleMuseumLogo.src === "https://cdn.shopify.com/s/files/1/0249/1083/t/3/assets/wm-logo-sm.png").toBeTruthy();
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/tests/Footer.test.tsx]: We will get better error messages if we do something like the below, and in next test as well 
```suggestion
    expect(whaleMuseumLogo.src).toBe("https://cdn.shopify.com/s/files/1/0249/1083/t/3/assets/wm-logo-sm.png");
```

File: WhaleSpotting/react-app/src/tests/Footer.test.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { render, screen } from "@testing-library/react";
+import { Footer } from "../components/Footer";
+import React from "react";
+
+test("renders Whale Museum logo in footer", () => {
+    render(<Footer />);
+
+    const whaleMuseumLogo = screen.getByTestId("whale-museum-logo");
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/tests/Footer.test.tsx]:
```suggestion
    const whaleMuseumLogo = screen.getByTestId("whale-museum-logo") as HTMLImageElement;
```

File: WhaleSpotting/react-app/src/styles/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
+@import "./Constants.scss";
+
+.footer {
+    position: fixed;
+    bottom: 0px;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Footer.scss]: Can be 0

File: WhaleSpotting/react-app/src/styles/Footer.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            color: $textOnPrimary;
+            font-family: $fontFamily;
+            text-decoration: none;
+
+            .logo {
+                height: 100%;
+            }
+
+            &.whale-link {
+                margin-left: 10%;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Footer.scss:30]: I think I prefer this without the left margin tbh, but happy to go with what you think if you prefer :)

File: WhaleSpotting/react-app/src/styles/Footer.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+@media (max-width: $mobileScreenSize) {
+    .footer {
+        height: $mobileFooterHeight;
+
+        .footer-logo-container {
+
+            .link {
+                display: block;
+                height: auto;
+                width: auto;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Footer.scss]: I can't see these doing anything?

File: WhaleSpotting/react-app/src/styles/Footer.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+                font-size: $smallMobileBodyFontSize;
+
+                .logo {
+                    display: block;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Footer.scss]: Can't see this doing anything either?

**Review by oskarwilliams — CHANGES_REQUESTED** [review]
As part of this can you please add a little more padding to the desktop home page at the bottom, seems to overrun a little

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**